rust-runtime/aws-smithy-http-server/src/response.rs

```diff
@@ -36,3 +36,7 @@ use crate::body::BoxBody;
 
 #[doc(hidden)]
 pub type Response<T = BoxBody> = http::Response<T>;
+
+pub trait IntoResponse<Protocol> {
```

Comment:
There's already a `#[doc(hidden)]` at the module level in `lib.rs`. `Response` is double hidden, which threw me off too - should I remove that?

Reply:
> There's already a #[doc(hidden)] at the module level in lib.rs

Ah, missed that. Thought it wasn't because `Response` was doc hidden.

Ok then, no need to double hide `Response` --- although, I'm thinking that middleware authors will need access to it right?

Reply:
I agree that library authors probably want to see `IntoResponse` and `FromRequest` eventually. But was thinking of keeping them hidden until everything in the RFC has been implemented before removing the `#[doc(hidden)]`. I'm not 100% sure on whether `IntoResponse` needs a second parameterization over the operation, which would be a breaking change.